rel/overlay/etc/default.ini

```diff
@@ -49,9 +49,12 @@ changes_doc_ids_optimization_threshold = 100
 ; applied conservatively. For example 1.0e+16 could be encoded as 1e16, so 4 used
 ; for size calculation instead of 7.
 ;max_document_size = 4294967296 ; bytes
-
 ; Maximum attachment size.
 ; max_attachment_size = infinity
+;
+; Whitelisted reserved document names (to allow modification of shard maps)
+; List of tuples of the form {dbname, [docid, ...]}
+;whitelist_doc_ids = [{"_dbs", ["_users", "_replicator", "_global_changes"]}]
```

Comment:
Technically dbs is already a configured parameter, `mem3.shards_db` I believe,  maybe just read `config:get("mem3", "shards_db", "_dbs")` when validating.

_replicator is a hard-coded maybe we don't need it to be configurable.  `_users` is from `chttpd_auth.authentication_db` so there can read from config and `_global_changes` I think is also hard coded like replicator.

Reply:
I want to see what @rnewson says here; I can certainly derive the values but he specifically said he wanted the list configurable. *shrug*

Reply:
I meant what nick said, that we fetch the names of these special databases from the existing config for each of them, sorry if I was ambiguous.

Reply:
I think using `SYSTEM_DATABASES` from `couch_db.hrl` is the best bet here, since we already define that centrally. Pushing a new version momentarily...